content/en/docs/instrumentation/python/_index.md

```diff
@@ -87,14 +69,22 @@ $ pip install -e ./opentelemetry-sdk
 
 ## Learn more
 
-- [API reference][]
+- [API & SDK reference][]
```

Comment:
These are all what we call "API reference" docs:

```suggestion
- [API reference][]
```

Reply:
If you agree to this change, you'll need to edit the markdown link definition below as well.